level_maker.cpp

```diff
@@ -2287,6 +2297,9 @@ PLevelMaker LevelMaker::topLevel(RandomGen& random, optional<CreatureFactory> fo
       case SettlementType::CAVE:
         queue = dragonCaveMaker(settlement);
         break;
+      case SettlementType::HUGE_CAVERN:
+        queue = hugeCavernMaker(settlement);
```

Comment:
Would be rather nice to specify the size of the SettlementType rather than it always being fixed.